### PR TITLE
Clarify email address field during setup

### PR DESCRIPTION
**Pull request by Pikachu920**
#### Description
Adds some helper text to the email field of the setup process.

#### Screenshot (if UI-related)
image

#### To-Dos

- [x] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

N/A

### CODE REVIEW

File: src/components/Login/JellyfinLogin.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const messages = defineMessages({
   password: 'Password',
   host: '{mediaServerName} URL',
   email: 'Email',
+  emailtooltip: 'This can be any valid email. It doesn\'t need to come from your {mediaServerName} instance.',
```
Comment (fallenbagel) [inline src/components/Login/JellyfinLogin.tsx]: Could you run prettier on this file?

**Review by Fallenbagel — CHANGES_REQUESTED** [review]
Needs prettier to be run on src/components/Login/JellyfinLogin.tsx

**Comment by Fallenbagel** [comment]
New look:
image

After hover:
image

**Review by Fallenbagel — APPROVED** [review]

**Comment by Fallenbagel** [comment]
@all-contributors please add @Pikachu920 for code

**Pull request merged**